docs/guides/bulk/download.md

```diff
@@ -0,0 +1,32 @@
+---
+heading: Bulk
+seo: Bulk | Bulk Download | How to Use Bulk Download
+title: Download
+description: How to use Bulk Download
+layout: sidebarleft
+apis: API Docs
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 2
+---
+
+# Bulk Download
+
+The Bulk download can be used to generate a csv or json file from a large number of records. Since the bulk download works asynchronously, two calls are required to download a file.
+
+### 1. Create a Bulk Job
+The `POST /bulk/query` creates a bulk job, which will build out a file in Cloud Elements. The records included in the file are based on the CEQL query used to generate the job. Since the job will take an unknown amount of time to complete, a callback url can also be included in the bulk job. When the job completes it will make a callback to the url. The query is composed of several parts:
+
+- Headers:
+  - `Elements-Async-Callback-Url`: The webhook to be notified when a job has completed
+- Query:
+  - `q` : A CEQL query for records to be included in the bulk job, i.e. `select * from contacts`
```

Comment:
in the bulk job, ~~i.e.~~ e.g., `select * from contacts`.

i.e. means "that is" and e.g. means "for example". Both should always be followed by a comma.
In general, it's frowned upon to use Latin abbreviations in tech writing because they can be confused and are trouble for non-native speakers. However, I think we're ok to continue using them for now.